docs/index.rst

```diff
@@ -208,6 +208,39 @@ steps:
          Creating test case test_login_1 for requirement: Login Example.
          Linking test case test_login_1 to requirement: Login Example.
 
+test-plan command
++++++++++++++++++
+
+The test-plan command allows to create either parent or child test plans. This
+is accomplished by passing or not the ``--parent-name`` option, if it is
+present then a child test plan will be created and linked with the specified
+parent and, if not specified, a parent test plan will be created.
+
```

Comment:
How about:

The test-plan command allows creating a parent test plan or child test plans.  This is done by using ``--parent-name`` option.

Create a parent test plan
    If ``parent-name`` option is not specified, then just a parent test plan will be created.
Create a child test plan
    If ``parent-name`` option is specified, then a child test plan will be created and linked to the specified parent test plan.